dashboard/config/courses/csd-2021.course

```diff
@@ -82,95 +50,40 @@
       }
     },
     {
-      "name": "CSD Implementation Guide",
-      "url": "https://docs.google.com/document/d/1u-eIlB2cAwMuSckzbMWkteQIcsXgez7FTrbjP8PBB4o/preview",
-      "key": "csd_implementation_guide",
-      "properties": {
-        "type": "Resource",
-        "audience": "Teacher"
-      },
-      "seeding_key": {
-        "resource.key": "csd_implementation_guide"
-      }
-    },
-    {
-      "name": "CSD Guide to Assessment",
-      "url": "https://docs.google.com/document/d/1PdGi9HCStRVDoIMKg2p5E4V4i_eN-CVqKgS-1VrZK18/preview",
-      "key": "csd_guide_to_assessment",
-      "properties": {
-      },
-      "seeding_key": {
-        "resource.key": "csd_guide_to_assessment"
-      }
-    },
-    {
-      "name": "CSD Guide to Differentiation",
-      "url": "https://docs.google.com/document/d/15Y6Mo3fMUtHq16SSskABgcYE-5xfk-Ewv0z_o4uaLdI/preview",
-      "key": "csd_guide_to_differentiation",
+      "name": "All Resources",
+      "url": "/courses/csd-2021/resources",
+      "key": "all_resources_1",
       "properties": {
+        "is_rollup": true
       },
       "seeding_key": {
-        "resource.key": "csd_guide_to_differentiation"
+        "resource.key": "all_resources_1"
       }
     },
     {
-      "name": "CSD Guide to Using Resources",
-      "url": "https://docs.google.com/document/d/1DUpyEfCI3AWk5YQ3CXIlP5WpWhydWE4hKUmOZMHnNRU/preview",
-      "key": "csd_guide_to_using_resources",
+      "name": "All Standards",
+      "url": "/courses/csd-2021/standards",
+      "key": "all_standards_1",
       "properties": {
+        "is_rollup": true
       },
       "seeding_key": {
-        "resource.key": "csd_guide_to_using_resources"
+        "resource.key": "all_standards_1"
       }
     },
     {
-      "name": "CSD Guide to Debugging",
-      "url": "https://docs.google.com/document/d/1-mXz53CAQ1dy3-RVGkoUnR0hfAwdWeza4kch6NAvLb8/preview",
-      "key": "csd_guide_to_debugging",
+      "name": "All Vocabulary",
+      "url": "/courses/csd-2021/vocab",
+      "key": "all_vocabulary_1",
       "properties": {
+        "is_rollup": true
       },
       "seeding_key": {
-        "resource.key": "csd_guide_to_debugging"
+        "resource.key": "all_vocabulary_1"
       }
     }
   ],
   "student_resources": [
-    {
```

Comment:
Did you mean to remove all the student resources?

Reply:
No... I see whats going on